packages/reports/addon/controllers/reports/report.js

```diff
@@ -110,6 +119,29 @@ export default class ReportsReportController extends Controller {
     }
   }
 
+  /**
+   * Updates the last added column
+   * @param {String} type - the added column type
+   * @param {Object} fragment - the added request fragment
+   * @param {String} fragment.id - fragment id
+   */
+  @action
+  updateLastAddedColumn(type, { id }) {
+    set(this, 'lastAddedColumn', { type, name: type === 'timeDimension' ? 'dateTime' : id });
```

Comment:
Any reason we don't just use the fragment here? Seems like an easier way to search because now we rely on searching backwards through the columns and having the newest one appended there. So if we wanted clone to add right after in the list it might not work

Reply:
The fragment is not yet created when we call `updateLastAddedColumn`.
It's a little on the awkward side because there's no guarantee it will be added successfully.. but if I queue `updateLastAddedColumn` _after_ adding, the `did-render` modifier (`setupElement`) is already called.

Reply:
If we wanted to clone right after we would need to change it to be index based or something similar